### PR TITLE
Created Map page with actual map

### PR DESCRIPTION
**Pull request by visitharri**

### CODE REVIEW

**Comment by visitharri** [comment]
My CSS is still scattered in too many places

File: frontend/src/pages/MapPage/MapPage.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const center: [number, number] = [61.494, 23.7798]; // Tampere
+    const zoom = 13;
+
+    return (
+        <div>
+            <MapContainer
+                center={center}
+                zoom={zoom}
+                scrollWheelZoom={true}
+                style={{ height: "100vh", width: "100%" }}
```
Comment (kemm0) [inline frontend/src/pages/MapPage/MapPage.tsx:15]: The styles should be removed from here and put into MapPage.scss.

File: frontend/src/pages/MapPage/MapPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import {render} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import MapPage from "./MapPage";
+
+let component = null;
+
+test("component loads", async () => {
+    component = render(<MapPage />);
+    expect(component).toBeTruthy();
```
Comment (kemm0) [inline frontend/src/pages/MapPage/MapPage.test.tsx:9]: Does this test that the component is visible? Can it be truthy even if it's not visible?

**Review by kemm0 — CHANGES_REQUESTED** [review]
Styles should be moved to a scss-file. Added one question about the test

File: frontend/src/pages/MapPage/MapPage.test.tsx (hunk 2 above)
Comment (visitharri) [reply frontend/src/pages/MapPage/MapPage.test.tsx]: Test has been updated now

File: frontend/src/pages/MapPage/MapPage.tsx (hunk 1 above)
Comment (visitharri) [reply frontend/src/pages/MapPage/MapPage.tsx]: Styles has been put into MapPage.scss

**Review by Mel-dev-hub — APPROVED** [review]
This one looks good to me, it's similar to the one I have at least.

**Comment by kemm0** [comment]
Look good 👍

**Review by kemm0 — APPROVED** [review]

**Comment by visitharri** [comment]
I am not sure, how issue #17 got linked to this, but that is absolutely a mistake. And #17 has nothing to do with #7, and should be unlinked.

**Pull request merged**